src/EPR.Accreditation.Facade/Controllers/AccreditationSiteMaterialController.cs

```diff
@@ -206,5 +206,31 @@ public class AccreditationSiteMaterialController : ControllerBase
 
             return Ok();
         }
+
+        [HttpGet("HasNpwdAccreditationNumber")]
+        public async Task<IActionResult> GetHasNpwdAccreditationNumber(
+            Guid accreditationExternalId,
+            Guid materialExternalId)
+        {
+            var hasNpwdAccreditationNumber = await _accreditationMaterialService.GetHasNpwdAccreditationNumber(
+                accreditationExternalId,
```

Comment:
Can we stop using the "external" bit please? Also refer to accreditationId as just id

Reply:
Changed